mcs/class/Makefile

```diff
@@ -50,6 +50,7 @@ endif
 
 mobile_common_dirs := \
 	corlib	\
+	Mono.Profiler.Log	\
```

Comment:
Also you need to move this down to at least below System.Core since it depends on that one. This is the cause of the errors:

```
error CS0006: Metadata file './../../class/lib/monodroid/System.dll' could not be found
error CS0006: Metadata file './../../class/lib/monodroid/System.Core.dll' could not be found
```

In fact, I'd just add it to the end of the list so the more "important" libraries are built first.